wsclient/src/test/java/org/hpccsystems/ws/client/WsDFUClientTest.java

```diff
@@ -378,4 +379,21 @@ public void getClustersTest() throws Exception
         assertTrue(clusters.stream().findFirst().isPresent());
     }
 
+    @Test
+    public void createSuperFileTest() throws Exception
+    {
+        try
+        {
+            wsdfuclient.createSuperfile("hpcc4j::tests::junit::mysuperfile", new String [] {""});
+        }
+        catch (Exception e)
```

Comment:
can the test exercise test both creating a nonexistent superfile and adding subfiles to an existing superfile, and trying to add a nonexistent subfile to check the error handling?